### ISSUE TITLE
Predict on Camvid Example With No Masks

### ISSUE THREAD

**Issue by ghost** [qubvel/segmentation_models.pytorch#607]
Newbie here, anyone willing to share code for making predictions on my test image without masks? Model is well trained, just struggling to code the dataloader so I can make predictions using test images only.

Thanks :)
